SoftLayer/CLI/custom_types.py

```diff
@@ -9,6 +9,7 @@
 import click
 
 
+# pylint: disable=inconsistent-return-statements
```

Comment:
try to disable  in the method convert as:
`def convert(self, value, param, ctx):  # pylint: disable=R1710`